inc/namespace.php

```diff
@@ -133,8 +133,11 @@ function get_config() {
  * Load the Object Cache dropin.
  */
 function load_object_cache() {
-	wp_using_ext_object_cache( true );
 	$config = get_config();
+	if ( ! $config['memcached'] && ! $config['redis'] ) {
```

Comment:
I think I would move these checks to where the functions are called in `load_platform()`. Looking at further now I'd probably also split the redis and memcached loading functions so that they just have one task each and don't need to check the config. That would make the functions behave the same way whenever they're called which will simplify any future unit tests.

Reply:
I've done this in e8046bc2df8b733c678d63abdeb16c93bb2a7afa.

To keep with back-compat and LTS policies, I've added `load_object_cache()` back in to trigger a notice in 30804921d875fd7ef5624a732fa51d532d6552d3.